packages/builder/src/components/automation/SetupPanel/AutomationBlockSetup.svelte

```diff
@@ -865,7 +865,7 @@
   <!-- Custom Layouts -->
   {#if stepLayouts[block.stepId]}
     {#each Object.keys(stepLayouts[block.stepId] || {}) as key}
-      {#if canShowField(key, stepLayouts[block.stepId].schema)}
+      {#if canShowField(stepLayouts[block.stepId].schema)}
```

Comment:
```ts
  function canShowField(value) {
    const dependsOn = value?.dependsOn
    return !dependsOn || !!inputData[dependsOn]
  }
```
this function used to have key and value parameters but key was removed, meaning all cases where this was being called with two arguments were broken